README.rst

```diff
@@ -5,14 +5,16 @@ Description |BuildStatus|_
 C connector. It is designed to allow you to connect to and query a
 **MySQL** database server using a simple API.
 
-Libdrizzle-redux in the sociomantic tsunami repository is a fork of the official
-**v5.1.4 Felmingham** at https://launchpad.net/libdrizzle.
+**Libdrizzle-redux** is a fork of the official **v5.1.4 Felmingham** release available at https://launchpad.net/libdrizzle.
```

Comment:
Yes, v5.1.5 wasn't released before dev on the the original project stopped, https://launchpad.net/libdrizzle/5.1